Stickdom/level1.js

```diff
@@ -28,6 +28,11 @@ class level1 extends Phaser.Scene
         'assets/fireball.png',
             {frameWidth: 17, frameHeight: 17}   
         );
+        this.load.spritesheet('boss', 
+            'assets/bigSpearman.png',
+            {frameWidth: 64, frameHeight: 100}
+            // { frameWidth: 32, frameHeight: 50 }
```

Comment:
- Used when drawing animations for sprite, sprite is scaled down
- Will remove when animations are complete